provisioning/build-tc.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
Might be worth changing to `#!/bin/bash` and add `set -ex` so that it halts on errors and the commands being run are shown in the TeamCity build log.